Pod/Classes/URBNAlertView.m

```diff
@@ -111,12 +119,14 @@ - (instancetype)initWithAlertConfig:(URBNAlertConfig *)config alertStyler:(URBNA
         NSDictionary *metrics = @{@"sectionMargin" : self.alertStyler.sectionVerticalMargin,
                                   @"btnH" : self.alertStyler.buttonHeight,
                                   @"lblHMargin" : self.alertStyler.labelHorizontalMargin,
-                                  @"titleVMargin" : titleMargin,
+                                  @"topVMargin" : @([titleMargin doubleValue] + [self.alertStyler.topMargin doubleValue]),
```

Comment:
@bobawhaler We will get this sorted out today. I apologize it is getting towards the peak time of the year for us so we are very busy with getting our apps ready.